app/src/pages/projects/ProjectsPage.tsx

```diff
@@ -24,17 +31,43 @@ import { ProjectsPageProjectsQuery } from "./__generated__/ProjectsPageProjectsQ
 import { ProjectsPageQuery } from "./__generated__/ProjectsPageQuery.graphql";
 import { ProjectActionMenu } from "./ProjectActionMenu";
 
-const REFRESH_INTERVAL_MS = 3000;
+const REFRESH_INTERVAL_MS = 10000;
+const LOOKBACK_DAYS = 7;
 
 export function ProjectsPage() {
+  const timeRange = useMemo(() => {
+    // Create a time range for the past 7 days
+    // Artificially set the end time to far in the future so that it is ostensibly is "current"
+    const now = Date.now();
+    return {
+      start: subDays(now, LOOKBACK_DAYS).toISOString(),
+      end: addDays(now, 365).toISOString(),
+    };
```

Comment:
@mikeldking Curious, is there benefit to making our time ranges optionally open?